### DISCUSSION TITLE
side-by-side only one time

### DISCUSSION THREAD

**Discussion by treatmesubj** [dandavison/delta#1665] [Q&A] [answered]
Hey, `delta` is the best.
Most of the time I prefer to not use `side-by-side`, but occasionally it provides a better view (for a screenshot, etc.) and I use it (by first temporarily un-commenting the option in `~/.gitconfig`, then running my command, then re-commenting the option in `~/.gitconfig`, which is not a good workflow)
I've read the [side-by-side docs](https://dandavison.github.io/delta/side-by-side-view.html) and see that I can use a _global_ environment variable to switch on `side-by-side` view like so, `export DELTA_FEATURES=+side-by-side`, and turn it off like so, `export DELTA_FEATURES=-side-by-side`.

Is there a clever way to add this functionality as an option to `git diff`?
It would be cool if I could use it like so: `git diff --staged --side-by-side`

I can't think of a way to implement it as a `git alias` in `~/.gitconfig` without losing all the `git diff` options

**Comment by dandavison** [comment, marked as answer]
Hi @treatmesubj, how about `git -c delta.side-by-side=true diff`?

**Reply by treatmesubj** [reply]
thanks @dandavison, alright, that's definitely better than what I was doing. Thank you!

**Reply by treatmesubj** [reply]
adding this `git alias` to `~/.gitconfig` seems to work. Feels decent
```
[alias]
    diff-sbs = -c delta.side-by-side=true diff
```
